src/rapidata/rapidata_client/order/rapidata_order.py

```diff
@@ -25,17 +25,29 @@ def __init__(
         openapi_service: OpenAPIService,
         name: str,
     ):
-        self.openapi_service = openapi_service
+        self._openapi_service = openapi_service
         self.order_id = order_id
         self._dataset = dataset
         self.name = name
         self._workflow_id = None
 
-    def submit(self):
+    def run(self, print_link: bool=True):
```

Comment:
im not sure how needed this parameter is. some functions have these kinds of parameters, maybe would be nice to mimic ML libraries and simply put a verbose=True parameter in some places.

Whats maybe a bigger change is that we should use logging not prints so its more configurable